SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialOSWorkerInterface.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/Connection/OutgoingMessages.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/SpatialLatencyTracer.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+class SPATIALGDK_API SpatialOSWorkerInterface
+{
+public:
+// 	FORCEINLINE bool IsConnected() { return bIsConnected; }
+
+	// Worker Connection Interface
+	virtual TArray<Worker_OpList*> GetOpList() PURE_VIRTUAL(AbstractSpatialWorkerConnection::GetOpList, return TArray<Worker_OpList*>(););
```

Comment:
@aleximprobable raised a question of whether we wanted to use the PURE_VIRTUAL macro here or just declare them as =0. I'm open to the discussion. I used PURE_VIRTUAL because I wanted to follow the Unreal style, but I'm not sure we gain much from it.

Reply:
I think we should use PURE_VIRTUAL to follow the Unreal style. I brought it up to mention that we don't have to implement functions we don't use for tests in derived classes. After a discussion, we agreed to have empty functions in derived test classes.

Reply:
As part of the discussion, I agreed to move the Mocks to a place they are more generically available/useful.